packages/components/data-table-manager/src/types.tsx

```diff
@@ -231,14 +231,15 @@ export type TColumnProps<Row extends TRow = TRow> = {
 };
 
 export type TDataTableManagerProps<Row extends TRow = TRow> = {
+  withRowSelection?: boolean;
   /**
    * Each object requires a unique `key` which should correspond to property key of
    * the items of `rows` that you want to render under this column, and a `label`
    * which defines the name shown on the header.
    * The list of columns to be rendered.
    * Each column can be customized (see properties below).
    */
-  columns: TColumnProps<Row>[];
+  columns?: TColumnProps<Row>[];
```

Comment:
I made this slight change because now we can also consume the data table without passing the props through the data table component but rather, through it's provider, migrating storybook to typescript actually helped me catch that we need to make the columns prop optional.